packages/beacon-node/src/chain/chain.ts

```diff
@@ -873,6 +873,36 @@ export class BeaconChain implements IBeaconChain {
     }
   }
 
+  /**
+   * Invalid state root error is critical and it causes the node to stale most of the time so we want to always
+   * persist preState, postState and block for further investigation.
+   */
+  async persistInvalidStateRoot(
+    preState: CachedBeaconStateAllForks,
+    postState: CachedBeaconStateAllForks,
+    block: SignedBeaconBlock
+  ): Promise<void> {
+    const blockSlot = block.message.slot;
+    const blockType = this.config.getForkTypes(blockSlot).SignedBeaconBlock;
+    const postStateRoot = postState.hashTreeRoot();
+    const suffix = `slot_${blockSlot}_invalid_state_root_${toRootHex(postStateRoot)}`;
```

Comment:
Noticed suffix is only used in a log, is that expected?
https://github.com/ChainSafe/lodestar/blob/b785bbdedfc90c8d586a79601cfa4a4f40175cd8/packages/beacon-node/src/chain/chain.ts#L1045

Reply:
Also suffix for pre state and post state should be `${suffix}_preState` and `${suffix}_postState` respectively